sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs

```diff
@@ -449,10 +450,7 @@ private void EnsureState()
                     Resource = Constants.Sas.Resource.BlobVersion;
                 }
             }
-            if (string.IsNullOrEmpty(Version))
-            {
-                Version = SasQueryParameters.DefaultSasVersion;
-            }
+            Version = SasQueryParameters.DefaultSasVersion;
```

Comment:
Should we edit docs for this to say it is a noop?

Reply:
Hm I might have to go find where I can go about documenting this. (or is it okay if I just update the changeLog)

Reply:
We should just edit the doc comment for the `Version` property that you put `[EditorBrowsable(Never)]` to add something along the lines of:

```C#
/// <remarks>
/// This property has been deprecated and we will always use the latest version
/// of the Storage service supported by the Azure.Storage.Blobs client library.  This
/// change does not have any impact on how your application generates or makes
/// use of SAS tokens.
/// </remarks>
```